beacon-chain/db/kv/migration_archived_index_test.go

```diff
@@ -66,9 +63,8 @@ func Test_migrateArchivedIndex(t *testing.T) {
 			eval: func(t *testing.T, db *bbolt.DB) {
 				err := db.View(func(tx *bbolt.Tx) error {
 					k := uint64(2048)
-					if v := tx.Bucket(stateSlotIndicesBucket).Get(bytesutil.Uint64ToBytesBigEndian(k)); !bytes.Equal(v, []byte("foo")) {
-						return fmt.Errorf("did not receive correct data for key %d, wanted 'foo' got %v", k, v)
-					}
+					v := tx.Bucket(stateSlotIndicesBucket).Get(bytesutil.Uint64ToBytesBigEndian(k))
+					assert.DeepEqual(t, []byte("foo"), v, fmt.Sprintf("did not receive correct data for key %d", k))
```

Comment:
```suggestion
					assert.DeepEqual(t, []byte("foo"), v, "Did not receive correct data for key %d", k)
```
Assertion library supports formatting.